### PR TITLE
[AdminUtils] Add channel create/rename/delete command

### LINKED ISSUE #19: [AdminUtils] Add delchannel and delrole
Please add two simple commands `delchannel` and `delrole` that can delete a channel or role based on name or channel/role ID. This is a really simple feature that could be combo'd with other cogs and automated triggers to do some cool stuff.

### PR DESCRIPTION
**Pull request by madebylydia**
This PR resolve #19.

### CODE REVIEW

File: adminutils/adminutils.py [hunk 1]
```diff
@@ -264,10 +283,66 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
...
+
+    @commands.group()
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_channels=True)
+    @commands.bot_has_permissions(manage_channels=True)
+    async def channel(self, ctx: commands.Context):
+        """Manage channels"""
+        pass
+
+    @channel.command(name="create", aliases=["add"])
```
Comment (fixator10) [inline adminutils/adminutils.py]: This allows user to create a channel in category, which user doesnt have access to.

For example, like category "Notifications" on screenshot:
yes

**Review by fixator10 — CHANGES_REQUESTED** [review]

File: adminutils/adminutils.py (hunk 1 above)
Comment (madebylydia) [reply adminutils/adminutils.py]: Should be fixed, I let you confirm me if it work now, and so I let you resolve this or not.

File: adminutils/adminutils.py [hunk 2]
```diff
@@ -264,10 +289,78 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
...
+    @commands.bot_has_permissions(manage_channels=True)
+    async def channel(self, ctx: commands.Context):
+        """Manage channels"""
+        pass
+
+    @channel.command(name="create", aliases=["add"], usage="[channel_type] [category] <name>")
+    async def channel_create(
+        self,
+        ctx: commands.Context,
+        channel_type: Optional[str] = "text",
```
Comment (fixator10) [inline adminutils/adminutils.py]: `str` will never be optional. 
https://discordpy.readthedocs.io/en/stable/ext/commands/commands.html#typing-optional

Can be fixed via function thats returns True if arg is in list, or via custom Converter class

File: adminutils/adminutils.py [hunk 3]
```diff
@@ -264,10 +289,78 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
...
+        Use double quotes if category has spaces
+
+        Examples:
+            `[p]channel add "The Zoo" awesome-channel` will create under the "The Zoo" category.
+            `[p]channel add awesome-channel` will create under no category, at the top.
+        """
+        self.check_channel_permission(ctx, category)
+        if channel_type not in ("text", "voice"):
+            raise commands.UserFeedbackCheckFailure(
+                "The channel's type can only be `voice` or `text`."
```
Comment (fixator10) [inline adminutils/adminutils.py]: i18n
```suggestion
                _("The channel's type can only be `voice` or `text`.")
```

File: adminutils/adminutils.py [hunk 4]
```diff
@@ -38,6 +38,24 @@ def cog_unload(self):
...
+        """
+        Check user's permission in a channel, to be sure he can edit it.
+        """
+        mc = channel_or_category.permissions_for(ctx.author).manage_channels
+        if mc:
+            return True
+        reason = (
+            "You are not allowed to edit this channel."
+            if not isinstance(channel_or_category, discord.CategoryChannel)
+            else "You are not allowed to edit in this category."
```
Comment (fixator10) [inline adminutils/adminutils.py]: i18n
```suggestion
            _("You are not allowed to edit this channel.")
            if not isinstance(channel_or_category, discord.CategoryChannel)
            else _("You are not allowed to edit in this category.")
```

File: adminutils/adminutils.py [hunk 5]
```diff
@@ -264,10 +289,78 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
...
+    async def channel(self, ctx: commands.Context):
+        """Manage channels"""
+        pass
+
+    @channel.command(name="create", aliases=["add"], usage="[channel_type] [category] <name>")
+    async def channel_create(
+        self,
+        ctx: commands.Context,
+        channel_type: Optional[str] = "text",
+        category: Optional[discord.CategoryChannel] = None,
```
Comment (fixator10) [inline adminutils/adminutils.py:319]: If category is not specified, it fails with `AttributeError: 'NoneType' object has no attribute 'permissions_for'`

**Review by fixator10 — CHANGES_REQUESTED** [review]

**Review by fixator10 — APPROVED** [review]

**Review by fixator10 — APPROVED** [review]

**Comment by fixator10** [comment]
https://github.com/fixator10/Fixator10-Cogs/commit/fc0d7c7841f77396ef341b1b10b72bacd12a846b: *commited a info.json to a wrong branch*

**Pull request merged**